resilience4j-spring/src/main/java/io/github/resilience4j/circuitbreaker/configure/CircuitBreakerConfiguration.java

```diff
@@ -61,4 +62,55 @@ public CircuitBreakerAspect circuitBreakerAspect(CircuitBreakerConfigurationProp
     public EventConsumerRegistry<CircuitBreakerEvent> eventConsumerRegistry() {
         return new DefaultEventConsumerRegistry<>();
     }
+    
+    /**
+     * Initializes the backends configured in the properties.
+     * 
+     * @param circuitBreakerRegistry The circuit breaker registry.
+     * @param eventConsumerRegistry The event consumer registry.
+     */
+    public void initializeBackends(CircuitBreakerRegistry circuitBreakerRegistry,
+    							   EventConsumerRegistry<CircuitBreakerEvent> eventConsumerRegistry) {
```

Comment:
EventConsumerRegistry<CircuitBreakerEvent> eventConsumerRegistry is unneeded dependency here